src/pika_conf.cc

```diff
@@ -149,6 +150,16 @@ int PikaConf::Load() {
   } else if (PIKA_SERVER_ID_MAX < std::stoull(server_id_)) {
     server_id_ = "PIKA_SERVER_ID_MAX";
   }
+  GetConfStr("run-id", &run_id_);
+  if (run_id_.empty()) {
+    char run_id[configRunIdSize + 1];
+    pstd::getRandomHexChars(run_id, configRunIdSize);
+    run_id_= std::string(run_id);
+  } else {
+    if (run_id_.length() != configRunIdSize) {
+      LOG(FATAL) << "run-id " << run_id_ << "is invalid, length should be " << configRunIdSize;
```

Comment:
`run_id xxxxxx is invalid, length should be 40`
Spaces are required before and after `run_id_`.